docs/css/extra.css

```diff
@@ -109,3 +109,18 @@ body[data-md-color-scheme="default"] .invertible img {
 body[data-md-color-scheme="slate"] .invertible img {
     filter: invert(100%) hue-rotate(180deg);
 }
+
+/* Rendered dataframe from jupyter */
+div.rendered_html {
+    width: 100% !important;
+    overflow-x: scroll;
```

Comment:
This creates scrollbars all over the place on my system:
![image](https://github.com/appliedAI-Initiative/pyDVL/assets/1548327/ccda606a-94d6-40b7-9da3-64f10a78b65d)

Removing it seems not to cause any problem. Is it necessary for you?

Reply:
Done.